subgraph.template.yaml

```diff
@@ -1,13 +1,12 @@
 specVersion: 0.0.4
 schema:
   file: ./schema.graphql
-# features:
-#   - grafting
-# graft:
-#  base: QmS4thuQDb5ndFecPrQWAAUoJkkjzjcWXugPBrRP7H2Cq8
-#  block: 106117601
+features:
+  - grafting
+graft:
```

Comment:
comment out pls

Reply:
Grafting is removed for Goerli and One via flags. 👍